File: Problem006.py
#25164150 in 0.0 seconds.
import time

start = time.time()

sum_of_squares = 0
square_of_sums = 0

max_num = 100

count = 0
for i in range(max_num+1):
    count += i
    sum_of_squares += i ** 2
square_of_sums = count ** 2

total = square_of_sums - sum_of_squares
end = time.time()
print(total,"in",end-start,"seconds.")

File: java/Problem006.java

public class Problem006 {

	public static void main(String args[]) {
		System.out.println(new Problem006().run());
	}
	
	public String run() {
		int sumOfSquares = 0;
		int squareOfSums = 0;
		int max = 100;
		
		int count = 0;
		for(int i = 0; i <= max; i++) {
			count += i;
			sumOfSquares += i * i;
		}
		squareOfSums = count * count;
		
		return "Answer: " + (squareOfSums - sumOfSquares);
	}
}
